source/background.js

```diff
@@ -1,7 +1,17 @@
+import addDomainPermissionToggle from 'webext-domain-permission-toggle';
 import browser from 'webextension-polyfill';
 import oneEvent from 'one-event';
 import optionsStorage from './options-storage.js';
 
+if (navigator.userAgent.includes('Firefox/')) {
+	// eslint-disable-next-line unicorn/prefer-top-level-await -- I specifically want to not stop the extension in case of errors
+	(async () => {
+		addDomainPermissionToggle({
+			title: 'Grant access to iframes',
```

Comment:
Since this issue is exclusive to Firefox, I'm limiting it to Firefox.

The current drawback of this setup is that it's kind of undocumented. I suppose a follow-up step would be to:

- include a "iframes detected, grant access. [docs link]" notification
- add to troubleshooting: https://ghosttext.fregante.com/troubleshooting/